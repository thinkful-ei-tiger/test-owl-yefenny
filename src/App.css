.App {
 display: flex; 
 font-family: 'Ubuntu', sans-serif; 
 height: 100vh; 
}