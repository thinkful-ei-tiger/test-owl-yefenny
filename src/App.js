import './App.css';
import ParticipantList from './participantList/ParticipantList';
import PARTICIPANTS from './dummy';
import Stage from './stage/Stage';
import Chat from './chat/Chat';

function App() {
  return (
    <div className='App'>
      <ParticipantList participants={PARTICIPANTS} />
      <Chat />
      <Stage />
    </div>
  );
}

export default App;
