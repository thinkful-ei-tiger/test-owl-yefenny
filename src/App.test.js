import React from 'react';
import ReactDom from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';

describe('App component', () => {
  it('renders wihout crashing', () => {
    const div = document.createElement('div');
    ReactDom.render(<App />, div);
    ReactDom.unmountComponentAtNode(div);
  });
  it('Render UI without problem', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
