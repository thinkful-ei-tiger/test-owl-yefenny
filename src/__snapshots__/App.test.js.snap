// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Render UI without problem 1`] = `
<div
  className="App"
>
  <div
    className="participant-list"
  >
    <ul>
      <li>
        <div
          className="participant"
        >
          <div
            className="participant-img"
          >
            <img
              alt="avatar"
              src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
            />
          </div>
          <div
            className="participant-details"
          >
            <h2>
              Koren Templeton
            </h2>
            <div
              className="participant-status"
            >
              <span
                className="circle bg-green"
              />
              <span
                className="status"
              >
                on stage
              </span>
            </div>
          </div>
        </div>
      </li>
      <li>
        <div
          className="participant"
        >
          <div
            className="participant-img"
          >
            <img
              alt="avatar"
              src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
            />
          </div>
          <div
            className="participant-details"
          >
            <h2>
              Caty Flucker
            </h2>
            <div
              className="participant-status"
            >
              <span
                className="circle bg-green"
              />
              <span
                className="status"
              >
                on stage
              </span>
            </div>
          </div>
        </div>
      </li>
      <li>
        <div
          className="participant"
        >
          <div
            className="participant-img"
          >
            <img
              alt="avatar"
              src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
            />
          </div>
          <div
            className="participant-details"
          >
            <h2>
              Axe Kubicka
            </h2>
            <div
              className="participant-status"
            >
              <span
                className="circle bg-green"
              />
              <span
                className="status"
              >
                on stage
              </span>
            </div>
          </div>
        </div>
      </li>
      <li>
        <div
          className="participant"
        >
          <div
            className="participant-img"
          >
            <img
              alt="avatar"
              src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
            />
          </div>
          <div
            className="participant-details"
          >
            <h2>
              Frank Runciman
            </h2>
            <div
              className="participant-status"
            >
              <span
                className="circle bg-green"
              />
              <span
                className="status"
              >
                in session
              </span>
            </div>
          </div>
        </div>
      </li>
      <li>
        <div
          className="participant"
        >
          <div
            className="participant-img"
          >
            <img
              alt="avatar"
              src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
            />
          </div>
          <div
            className="participant-details"
          >
            <h2>
              Ashla Attwool
            </h2>
            <div
              className="participant-status"
            >
              <span
                className="circle bg-green"
              />
              <span
                className="status"
              >
                on stage
              </span>
            </div>
          </div>
        </div>
      </li>
    </ul>
  </div>
  <div>
    <div
      className="event"
    >
      <div
        className="message-type"
      >
        <div
          className="message-avatar"
        >
          <img
            alt="profile"
            src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
          />
        </div>
        <div
          className="message-information"
        >
          <div
            className="message-header"
          >
            <h2>
              Axe Kubicka
            </h2>
            <span>
              07:50 a.m.
            </span>
          </div>
          <div
            className="message"
          >
            <p>
              Hello world
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Caty Flucker
          </strong>
           gave a thumbs up üëç
        </span>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Frank Runciman
          </strong>
           gave a thumbs down üëé
        </span>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Koren Templeton
          </strong>
           raise their hand ‚úã
        </span>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Koren Templeton
          </strong>
           clapped üëè
        </span>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Koren Templeton
          </strong>
           joined
        </span>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Ashla Attwool
          </strong>
           left
        </span>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Axe Kubicka
          </strong>
           joined the stage
        </span>
      </div>
    </div>
    <div
      className="event"
    >
      <div>
        <span>
          <strong>
            Axe Kubicka
          </strong>
           left the stage
        </span>
      </div>
    </div>
  </div>
  <div
    className="stage"
  >
    <div
      className="partners"
    >
      <div
        className="partner"
      >
        <div
          className="header"
        >
          <h2>
            Caty Flucker
          </h2>
          <div
            className="icons"
          >
            <span>
              <img
                alt="mute"
                src="volume.svg"
              />
              <img
                alt="search"
                src="search.svg"
              />
            </span>
          </div>
        </div>
        <div
          className="avatar"
        >
          <img
            alt="profile"
            src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
          />
        </div>
      </div>
      <div
        className="partner"
      >
        <div
          className="header"
        >
          <h2>
            Axe Kubicka
          </h2>
          <div
            className="icons"
          >
            <span>
              <img
                alt="mute"
                src="volume.svg"
              />
              <img
                alt="search"
                src="search.svg"
              />
            </span>
          </div>
        </div>
        <div
          className="avatar"
        >
          <img
            alt="profile"
            src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
          />
        </div>
      </div>
      <div
        className="partner"
      >
        <div
          className="header"
        >
          <h2>
            Ashla Attwool
          </h2>
          <div
            className="icons"
          >
            <span>
              <img
                alt="mute"
                src="volume.svg"
              />
              <img
                alt="search"
                src="search.svg"
              />
            </span>
          </div>
        </div>
        <div
          className="avatar"
        >
          <img
            alt="profile"
            src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
          />
        </div>
      </div>
    </div>
    <div
      className="personal-stage"
    >
      <div
        className="personal-img"
      >
        <img
          alt="personal"
          src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
        />
      </div>
    </div>
  </div>
</div>
`;
